### ISSUE TITLE
TypeError: can't convert return value to desired type

### ISSUE THREAD

**Issue by c10ud** [emesene/emesene#327]
If you fix this, we can release somekind of emesene in debian/ubuntu.

Source: unknown (emesene code, not libs)
Doesn't show a line number and/or traceback.

**Comment by Sbte** [comment]
I tried several debugging facilities of which none could hold on long enough to find function calls close to the error.

Easiest way to replicate is by opening a new chat window.

I'll see if I can add more debugging to trace down this problem. And if that could all get in it would be nice for future use too.

**Comment by c10ud** [comment]
i hate this bug. really, really hate it. i tried with some python debug stuff but it's not caught. but it excepts because a test-development version of ubuntu raises the "collecting bug information"..but doesn't complete its job because the package is not in the archives. I am suspecting this might be the result of some 

> > this_is_an_integer_var = 3
> > this_is_an_integer_var += this_method_returns_a_string()

someone with l33t debugging skills might be able to know the right tools for the job. i'll keep asking google atm

**Comment by c10ud** [comment]
i might have found the offending line

gui/gtkui/Conversation.py @ ~90

> ```
>    self.output = OutputText(self.session.config)
> ```

now, i'll dig into that

**Comment by c10ud** [comment]
AdiumTextBox.py - OutputView - clear() method

**Comment by c10ud** [comment]
```
    self.load_string(body,
            "text/html", "utf-8", "file://%s" % self.theme.path)
```

**Comment by c10ud** [comment]
seems like a bug in the adium/webkit themes (!)

**Comment by c10ud** [comment]
it only triggers with the on_navigation_requested(self, widget, WebKitWebFrame, WebKitNetworkRequest) callback enabled (!!!)

**Comment by c10ud** [comment]
FUUUUUUUUUUUUUUUUUUUUUUU-

closed by 1e424e7619f6bbb75e8f5172c434c6c7f819b904
see http://code.google.com/p/pywebkitgtk/issues/detail?id=2#c1

**Comment by Sbte** [comment]
Awesome debugging skills :P
